src/noos_inv/docker.py

```diff
@@ -1,31 +1,33 @@
 import os
 from typing import Optional
 
-from invoke import Collection, Context, task
+from invoke import Collection, Config, Context, task
 
 from . import utils
 
 
-CONFIG = {
-    "docker": {
-        # Sensitive
-        "repo": None,
-        "user": "AWS",
-        "token": None,
-        "arg": None,
-        # Non-sensitive
-        "file": "Dockerfile",
-        "context": ".",
-        "name": "webserver",
-        "tag": "test",
+CONFIG = Config(
+    defaults={
+        "docker": {
+            # Sensitive
+            "repo": None,
+            "user": "AWS",
+            "token": None,
+            "arg": None,
+            # Non-sensitive
+            "file": "Dockerfile",
+            "context": ".",
+            "name": "webserver",
+            "tag": "test",
+        }
     }
-}
+)
 
 
 # Docker deployment workflow:
 
 
-@task
+@task()
```

Comment:
💡 

Does initiating the decorator (required for base-class declared decorators) solve the `mypy` linting issue?

Reply:
Yes it does